docs/design/gatewayapi-translator.md

```diff
@@ -172,28 +139,32 @@ spec:
 
 ## Computing Status
 
-Gateway API specifies a rich set of status fields & conditions for each resource.
-To be conformant, Envoy Gateway needs to compute the appropriate status fields and conditions as it's processing resources.
+Gateway API specifies a rich set of status fields & conditions for each resource. To achieve conformance, Envoy Gateway
+must compute the appropriate status fields and conditions for managed resources.
 
-Status needs to be computed and set for:
+Status is computed and set for:
 
-- the GatewayClass (gatewayclass.status.conditions)
-- each Listener for each Gateway (gateway.status.listeners)
-- each Gateway, based on its Listeners' statuses (gateway.status.conditions)
-- each ParentRef for each Route (route.status.parents)
+- The managed GatewayClass (`gatewayclass.status.conditions`).
+- Each managed Gateway, based on its Listeners' status (`gateway.status.conditions`). For the Kubernetes provider, the
+  Envoy Deployment and Service status are also included to calculate Gateway status.
+- Listeners for each Gateway (`gateway.status.listeners`).
+- The ParentRef for each Route (`route.status.parents`).
 
-The Gateway API translator will take the approach of populating status on the resources themselves as they're being processed, and then passing those statuses off to another component to persist the updates to the Kubernetes API or other backend.
+The Gateway API translator is responsible for calculating status conditions while translating Gateway API resources to
+the IR and publishing status over the [message bus][]. The Status Manager subscribes to these status messages and
```

Comment:
message bus needs a link

Reply:
@arkodg the link is in [L251](https://github.com/envoyproxy/gateway/pull/645/files#diff-9525aca7f46ffbe1952dae4f2c5be610a6048e8c65af05dd651858ac3ce1e16fR251). I generated the docs, ran them locally, and verified the link to `watching.md` works.